### PR TITLE
fixed get-dash page

### PR DESCRIPTION
**Pull request by mrleewong**

### CODE REVIEW

File: src/js/tables.js [hunk 1]
```diff
@@ -49,13 +49,16 @@
 				$('[data-name]').each(function() {
 					var that = $(this),
```
Comment (stpdenis) [inline src/js/tables.js]: Why that, not self? Shouldn't it be Jq object already, you're iterating array of jq objects?

File: src/js/tables.js [hunk 2]
```diff
@@ -49,13 +49,16 @@
...
 						exchangeName = that.data('name'),
+						$result = that.find('[data-price="result"]'),
+						resultName;
+					if (resultExchanges) {
 						resultName = resultExchanges.filter(function(el) {
 							if (el) {
```
Comment (stpdenis) [inline src/js/tables.js]: return el && el.exchange === exchangeName

**Comment by mrleewong** [comment]
@stpdenis fixed

File: src/js/tables.js [hunk 3]
```diff
@@ -47,18 +47,21 @@
 					price = 'price_' + currency;
 				}
 				$('[data-name]').each(function() {
-					var that = $(this),
-						exchangeName = that.data('name'),
+					var self = $(this),
```
Comment (stpdenis) [inline src/js/tables.js]: just this, this is already jq object

**Pull request merged**